test/tasks_fine_tuning_task_test.py

```diff
@@ -18,10 +19,16 @@ class TestFineTuningTask(unittest.TestCase):
     def _compare_model_state(self, state_1, state_2, check_heads=True):
         return compare_model_state(self, state_1, state_2, check_heads=check_heads)
 
-    def _get_fine_tuning_config(self, head_num_classes=1000):
+    def _get_fine_tuning_config(
+        self, head_num_classes=1000, pretrained_checkpoint=False
+    ):
         config = get_fast_test_task_config(head_num_classes=head_num_classes)
         config["name"] = "fine_tuning"
         config["num_epochs"] = 10
+
+        if pretrained_checkpoint:
+            config["pretrained_checkpoint"] = ""
```

Comment:
Let's add a dummy string here so that it's clear what the intention is. Otherwise this looks like a test case for when the path isn't provided.
```
            config["pretrained_checkpoint"] = "path/to/checkpoint"
```